tools/cnvkit_batch_only.cwl

```diff
@@ -0,0 +1,63 @@
+cwlVersion: v1.0
```

Comment:
Is this just a copy and that's why you're keeping the 1.0 version?

Reply:
Yeah, it was a copy, I can make it 1.2 for the benefit of mankind